Cocktails9/Cocktails9/Preview Content/DrinksGrid.swift

```diff
@@ -33,6 +29,7 @@ struct LandmarkGrid: View {
                 .onAppear {
                     Task {
                         await networkManager.fetchDrinks()
+                        user = UserManager.loadUser()
```

Comment:
you don't need to `await` for api in order to load user. Those are 2 separate sources.
```suggestion
                        user = UserManager.loadUser()
                        await networkManager.fetchDrinks()
```